src/lexer.d

```diff
@@ -88,10 +89,10 @@ version (unittest)
 
 /***********************************************************
  */
-extern (C++) class Lexer
+class Lexer
```

Comment:
Apparently ``idPool`` got moved to ``Identifier``.  So it is just that one method then.
I guess I could raise a PR to move ``isValidIdentifier`` to the ``Identifier`` class too.